Makefile

```diff
@@ -3,29 +3,29 @@ TRAVIS_BUILD_NUMBER?=debug0
 
 .PHONY: coverage get test clean
 
-all: get coverage AWSnycast
+all: _vendor coverage AWSnycast
 
-AWSnycast: *.go */*.go
-	go build -a -tags netgo -ldflags '-w' .
-	strip AWSnycast
+_vendor: Gomfile
+	gom install
 
-test:
-	go test -short ./...
+AWSnycast: *.go */*.go _vendor
+	gom build -a -tags netgo -ldflags '-w' .
+	strip AWSnycast
```

Comment:
indeed I remember getting the error for failed stripped symbols or something similar, in all cases i think moving it to the packaging step is sane.